### DISCUSSION TITLE
Migrate Users Auth from Firebase to Supabase

### DISCUSSION THREAD

**Discussion by idrissdjio** [supabase/supabase#20224] [Questions]
Greetings,

I successfully transferred user data from Firebase Authentication to Supabase, following the guidelines provided in the documentation at the following link: https://supabase.com/docs/guides/resources/migrating-to-supabase/firebase-auth

While the user details have been migrated successfully, Supabase is displaying a status of "awaiting verification." It's worth noting that I have disabled email confirmation on Supabase. Could you kindly assist me with resolving this issue?

My goal is to enable users to log in seamlessly without the necessity of going through the email verification process.

Thank you for your assistance.

**Comment by GaryAustin1** [comment]
Without reviewing that process, I'm going to assume you have an email and password set for each user.  If you set a valid date/time into the email_confirmed_at column in the auth.users table that should verify the user.
![image](https://github.com/supabase/supabase/assets/54564956/c603b534-9d6c-4319-b3df-45a2ed11e4ad)

Please try it on a single user and see if that works for you before running sql against all your roles.

**Comment by idrissdjio** [comment]
![Screenshot (18)](https://github.com/supabase/supabase/assets/57317425/bfdf2d76-a18c-4002-9fb3-f22a7320f55b)

Thank you for your feedback. I am new to supabase and I used the default authentication system with email and password. What do you recommend at this stage?

**Reply by GaryAustin1** [reply]
I've never used the Firebase migration code and it looks to be about 2 years since last update so it is possible some fields in auth.users are new since then.  Your best bet to be safe is to generate an issue in the library used to import things.

Did you test a user by adding an email_confirmed_at date to the column with SQL?
If that does not let you sign in then you could also put a date/time in a last_sign_in_at column but I don't know if that prevents you from signing in or not.  It may just work and change when they actually sign in.
![image](https://github.com/supabase/supabase/assets/54564956/a0f34cbb-ea37-4b61-b30a-bac2e369e359)

**Reply by GaryAustin1** [reply]
And this all assumes you were able to get the password hashes over as I'm pretty sure is discussed in the migration guide.

**Comment by idrissdjio** [comment]
Thank you for your prompt feedack, I will keep debugging and will generate an issue in the library.
From your table, I can see that you customized your users table by yourself, I will try this alternative here too.

**Reply by GaryAustin1** [reply]
NO... don't touch the auth schema as far as changing types or adding columns.
It is not a customized user table I'm showing.
It is THE auth.users table in the auth schema that Supabase uses for auth.
The screen you are looking in your case is just a summary of the real table in the auth UI of the dashboard.
To see the real table you use the table UI tab, select auth schema and then select the users table.
